dolphinscheduler-ui/src/common/common.ts

```diff
@@ -361,14 +360,6 @@ export const workflowExecutionState = (
     isSpin: false,
     classNames: 'success'
   },
-  DELAY_EXECUTION: {
-    id: 12,
-    desc: `${t('project.workflow.delay_execution')}`,
```

Comment:
You should remove these i18n too.

Reply:
> You should remove these i18n too.

![image](https://github.com/user-attachments/assets/6cd62317-8fdb-4c51-aa3f-cad0a8773529)
This is referenced by Task Instance

Reply:
The others were not used.

Reply:
> The others were not used.

done